GameDataParser/Parsers/ItemStatsParser.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Xml;
+using GameDataParser.Crypto.Common;
+using GameDataParser.Files;
+using Maple2Storage.Enums;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    public class ItemStatsParser : Exporter<List<ItemStatsMetadata>>
+    {
+        public ItemStatsParser(MetadataResources resources) : base(resources, "item-stats") { }
+
+        protected override List<ItemStatsMetadata> Parse()
+        {
+            List<ItemStatsMetadata> items = new List<ItemStatsMetadata>();
+            foreach (PackFileEntry entry in Resources.XmlFiles)
+            {
+                if (!entry.Name.StartsWith("itemoption/constant") && !entry.Name.StartsWith("itemoption/option/random") && !entry.Name.StartsWith("itemoption/option/static"))
+                {
+                    continue;
+                }
+                if (entry.Name.Contains("petequipment.xml") || entry.Name.Contains("weaponmanual.xml") || entry.Name.Contains("armormanual.xml") || entry.Name.Contains("accmanual.xml") || entry.Name.Contains("etc.xml") || entry.Name.Contains("gemstone.xml") || entry.Name.Contains("mergematerial.xml") || entry.Name.Contains("skin.xml"))
+                {
+                    continue;
+                }
+
+                XmlDocument innerDocument = Resources.XmlMemFile.GetDocument(entry.FileHeader);
+                XmlNodeList nodeList = innerDocument.SelectNodes("/ms2/option");
+                string filename = Path.GetFileNameWithoutExtension(entry.Name);
+
+                foreach (XmlNode node in nodeList)
+                {
+                    ItemStatsMetadata metadata = new ItemStatsMetadata();
+                    ItemOptions optionsMetadata = new ItemOptions();
+                    List<Stat> statMetadata = new List<Stat>();
+                    int itemId = string.IsNullOrEmpty(node.Attributes["code"]?.Value) ? 0 : int.Parse(node.Attributes["code"].Value);
+                    metadata.ItemId = itemId;
+
+                    byte grade = string.IsNullOrEmpty(node.Attributes["grade"]?.Value) ? 0 : byte.Parse(node.Attributes["grade"].Value);
+                    optionsMetadata.Grade = grade;
+
+                    if (!string.IsNullOrEmpty(node.Attributes["optionNumPick"]?.Value))
+                    {
+                        List<string> list = new List<string>(node.Attributes["optionNumPick"].Value.Split(","));
+                        optionsMetadata.OptionNumPick = byte.Parse(list[0]);
+                    }
+
+                    // float abp = string.IsNullOrEmpty(node.Attributes["abp_rate_base"]?.Value) ? 0 : float.Parse(node.Attributes["abp_rate_base"].Value);
```

Comment:
Because I don't know what Attributes it's referring to. I left commented for future people to be able to use if someone figured out what these attributes are.